app/src/utils/data/contributions.ts

```diff
@@ -7,6 +7,8 @@ import {
   GrantMetadata,
   GrantRound,
   GrantRoundMetadata,
+  GrantRoundDonation,
+  MetaPtr,
```

Comment:
(random comment location for threading)

Clicking the "Gitcoin Building Gitcoin" link takes me to the home page, but I'd expect it to take me to the round page. You can see in this screenshot that it shows the homepage URL on hover. It seems this doesn't work in other places that leverage the same component, so this slipped by previously it seems

![image](https://user-images.githubusercontent.com/17163988/146278281-2501336f-e16e-4b3e-aea8-a3dcc398eb8d.png)

Reply:
this one is more difficult to do than it seems, since we are using the `ContributionDetail.vue` component for both matchingPool contributions as well as grant contributions - we need a different link for each, I guess we could make create a link field inside ContributionsDetail

Reply:
If you want to create a separate issue to track this and handle it in a separate PR, I think that's fine, cc @apbendi

Reply:
#597